templates/General/ocp-performance.jsonnet

```diff
@@ -184,6 +184,49 @@ local stackroxMem = genericGraphLegendPanel('Top 25 stackrox container RSS bytes
   )
 );
 
+// Dynatrace
```

Comment:
@krishvoor As discussed, let keep this Hypershift related metrics in to hypershift-performance dash.

Reply:
@mukrishn I made the changes as follows:
1) OneAgent Pods Will be captured per cluster serving nodes
2) ActiveGate  & Opentelemetry Pods will be captured at the Management Cluster metrics level